refinery/data_set_manager/search_indexes.py

```diff
@@ -53,14 +55,21 @@ def index_queryset(self, using=None):
     # https://groups.google.com/forum/?fromgroups#!topic/django-haystack/g39QjTkN-Yg
     # http://stackoverflow.com/questions/7399871/django-haystack-sort-results-by-title
     def prepare(self, object):
+
         data = super(NodeIndex, self).prepare(object)
         annotations = AnnotatedNode.objects.filter(node=object)
-        uuid = str(object.study.id)
+        suffix = str(object.study.id)
+
+        try:
+            data_set = get_data_set_for_study_uuid(object.study.uuid)
+            data['data_set_uuid'] = data_set.uuid
+        except RuntimeError as e:
+            logger.warn(e)
```

Comment:
I hit this error: Thinking about it, I guess other node types don't have data_sets: Perhaps check the node type first, and only try to add data_set_uuid if there actually should be one?